config/testgrids/kubernetes/sig-cluster-lifecycle/config.yaml

```diff
@@ -18,7 +18,7 @@ dashboard_groups:
     - sig-cluster-lifecycle-cluster-api-provider-openstack
     - sig-cluster-lifecycle-cluster-api-provider-cloudstack
     - sig-cluster-lifecycle-cluster-api-provider-nested
-    - sig-cluster-lifecycle-cluster-api-operator-0.15
+    - sig-cluster-lifecycle-cluster-api-operator-0.17
```

Comment:
This is all possible directly on the jobs
https://github.com/kubernetes/test-infra/blob/master/testgrid/config.md#prow-job-configuration

Reply:
@BenTheElder it is already configured in our jobs: https://github.com/kubernetes/test-infra/blob/master/config/jobs/kubernetes-sigs/cluster-api-operator/cluster-api-operator-periodics-release-0-17.yaml#L26. Are you suggesting that it is enough to have it as an annotation as we already have now and we can drop it from config.yaml?